src/Components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './NavBar.css'
+
+function NavBar({ showMenu, setShowMenu }) {
+
+    return (
+        <nav className="nav-bar">
+            <NavLink to="/" className="logo">
+                DOVE
+            </NavLink>
+            <div className="right-nav">
+                <ul>
+                    <NavLink to="/" className="mobile-hide">Home</NavLink>
```

Comment:
These `<NavLink>`s should be wrapped with `<li>`

Reply:
Thanks! Will fix this.